### ISSUE TITLE
[tumblr] Retrieving posts by searching for URL is broken?

### ISSUE THREAD

**Issue by kr428** [friendica/friendica#14346]
So far, I was able in Friendica to retrieve a random Tumblr post by posting its url, like

https://www.tumblr.com/mini-moss/739764774537134080/indigo-nights-twitter-instagram-bluesky?source=share

... to the search bar and hit "search". On an instance running a recent 2024.06RC, this isn't working anymore all of a sudden; by then, clicking "search" after entering such an URL I get different behaviours ranging from a plan "Forbidden" page to "Not found / contact not found" page in my Friendica instance.

Somehow related to #14303 ?

**Comment by annando** [comment]
Try it without the `?source=share`.

**Comment by AlfredSK** [comment]
Indeed, the forbidden error could be related to the `%3f` issue because of the ? in the URL.

**Comment by kr428** [comment]
> Try it without the `?source=share`.

Yeah, tried that too, caused a "not found" for that particular object.

**Comment by annando** [comment]
There are some Tumblr posts from restricted accounts that cannot be fetched. Best is to check with other posts from other accounts as well.

**Comment by kr428** [comment]
> There are some Tumblr posts from restricted accounts that cannot be fetched. Best is to check with other posts from other accounts as well.

Had that in mind too but it doesn't seem to be the case at least for the link posted above - this opens well also in a private tab without being logged in to Tumblr but the Friendlca search says "not found". After playing with some other Tumblr accounts, I'm not seeing a consistent pattern here.

**Comment by kr428** [comment]
... and even this way it seems to vary, depending on whatever: This post...

https://www.tumblr.com/russiawave/637302046043832320?source=share

... looked up on loma.ml returned me to the post, on there, exactly once. Now, when searching for that particular post, I always end up on the Friendica display of the _profile_ which created this post, while the post itself doesn't seem available.
